test/unit/lib/plugins/aws/package/lib/mergeIamTemplates.test.js

```diff
@@ -282,6 +282,29 @@ describe('lib/plugins/aws/package/lib/mergeIamTemplates.test.js', () => {
         service = fixtureData.serviceConfig.service;
       });
 
+      it('should support `provider.iam.role.name`', async () => {
+        const customRoleName = 'custom-default-role';
+        const result = await runServerless({
+          fixture: 'function',
+          cliArgs: ['package'],
+          configExt: {
+            provider: {
+              iam: {
+                role: {
+                  name: customRoleName,
+                },
+              },
+            },
+          },
+        });
+
+        const { cfTemplate, awsNaming } = result;
+        const iamRoleLambdaResource = cfTemplate.Resources[awsNaming.getRoleLogicalId()];
+        const roleName = awsNaming.getRoleName();
+        expect(roleName).to.be.eq(customRoleName);
```

Comment:
let's skip this check as it's already tested as a part of `naming.test.js` tests

Reply:
:white_check_mark: